assets/src/latest-stories-block/block/edit.css

```diff
@@ -0,0 +1,316 @@
+/* Placeholder styles */
+.wp-block-web-stories-latest-stories.placeholder {
```

Comment:
Can we have the same selectors/markup as there is on the frontend if not exact same but for the common layout it should be same. This could reduce the editor specific styles we need to write all the time.

Reply:
Addressed. Replaced root selector to `web-stories` from `wp-block-web-stories-latest-stories`.